src/styles/blocks.scss/card.scss

```diff
@@ -0,0 +1,83 @@
+$base-margin: 0;
+$base-font: roboto, helvetica, sans-serif;
+$text-color: #616070;
+$blue-color: #00acdc;
+
+.card {
+  flex-direction: column;
+  display: flex;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px;
+  max-width: 200px;
+
+  &__image {
+    padding-top: 32px;
+    width: 160px;
+    height: 134px;
+    margin: 0 3px;
+  }
+
+  &__title {
+    margin: $base-margin;
+    padding-top: 40px;
+    font-family: $base-font;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__product-code {
+    margin: $base-margin;
+    padding-top: 5px;
+    font-family: $base-font;
+    font-size: 10px;
+    font-weight: 400;
+    color: $text-color;
+  }
+
+  &__price-amount {
+    margin: $base-margin;
+    font-family: $base-font;
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__price-title {
+    margin: $base-margin;
+    font-family: $base-font;
+    font-size: 12px;
+    color: $text-color;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: $base-margin;
+    padding-top: 24px;
+  }
+
+  &__buy-link {
+    display: inline-block;
+    height: 40px;
+    width: 166px;
+    line-height: 40px;
+    margin: 16px 0 15px;
+    border-radius: 5px;
+    background-color: $blue-color;
+    color: white;
+    font-family: $base-font;
+    font-size: 14px;
+    font-weight: 700;
+    text-decoration: none;
+    text-align: center;
+  }
+
+  &__buy-link:hover {
+    border: 1px solid $blue-color;
```

Comment:
add a transparent border on this element, and change only color on hover to fix it (button part):

![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/b7a9ef27-ded1-465f-9135-022c8699be02)

Also, try to fix stars position to pass tests